homeassistant/components/matter/sensor.py

```diff
@@ -1488,4 +1488,28 @@ def _update_from_device(self) -> None:
         entity_class=MatterSensor,
         required_attributes=(clusters.ServiceArea.Attributes.EstimatedEndTime,),
     ),
+    MatterDiscoverySchema(
+        platform=Platform.SENSOR,
+        entity_description=MatterSensorEntityDescription(
+            key="DoorLockDoorOpenEvents",
+            translation_key="door_lock_door_open_events",
```

Comment:
The translation key might be a bit wordy with `door_lock_door`. I think we could just drop the `door_lock` part there? So just have `door_open_events`? The string for this is just "Door opened events", not including the door lock.